### PR TITLE
Bind String Contains Operator Values

### PR DESCRIPTION
**Pull request by proggeramlug**
String contains operator values (`~~`, `!~`, `~=`, etc) are now correctly bound as query parameters for SQL databases (#140).

### CODE REVIEW

File: Sources/FluentSQL/SQLQueryConverter.swift [hunk 1]
```diff
@@ -210,11 +210,11 @@ public struct SQLQueryConverter {
                 let right: SQLExpression
                 switch method {
                 case .anywhere:
-                    right = SQLRaw("%" + string.description + "%")
+                    right = SQLRaw("\"%" + string.description + "%\"")
```
Comment (tanner0101) [inline Sources/FluentSQL/SQLQueryConverter.swift]: Using `SQLLiteral.string` here instead of `SQLRaw` will add quotes appropriate for each DB.

Comment (proggeramlug) [reply Sources/FluentSQL/SQLQueryConverter.swift]: perfect, that's what I was looking for!

Comment (gwynne) [reply Sources/FluentSQL/SQLQueryConverter.swift]: Per discussion in Discord, use `right = SQLBind("%\(string)%")`

File: Sources/FluentSQL/SQLQueryConverter.swift [hunk 2]
```diff
@@ -210,11 +210,11 @@ public struct SQLQueryConverter {
...
                 case .prefix:
-                    right = SQLRaw(string.description + "%")
+                    right = SQLRaw("\"" + string.description + "%\"")
```
Comment (gwynne) [inline Sources/FluentSQL/SQLQueryConverter.swift]: Per discussion in Discord, use `right = SQLBind("\(string)%")`

File: Sources/FluentSQL/SQLQueryConverter.swift [hunk 3]
```diff
@@ -210,11 +210,11 @@ public struct SQLQueryConverter {
...
                 case .suffix:
-                    right = SQLRaw("%" + string.description)
+                    right = SQLRaw("\"%" + string.description + "\"")
```
Comment (gwynne) [inline Sources/FluentSQL/SQLQueryConverter.swift]: Per discussion in Discord, use `right = SQLBind("%\(string)")`

**Comment by tanner0101** [comment]
These changes are now available in [1.0.0-beta.3.3](https://github.com/vapor/fluent-kit/releases/tag/1.0.0-beta.3.3)

**Pull request merged**